sys-apps/mouseemu/mouseemu-0.15-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit edo toolchain-funcs
+
+DESCRIPTION="Emulates scrollwheel, right- & left-click for one-button mice/touchpads"
+HOMEPAGE="http://geekounet.org/powerbook/"
+SRC_URI="mirror://gentoo/${P}.tar.gz"
```

Comment:
Same problem here, we can't rely on mirror. Maybe last-rite instead?

Reply:
Fine with me.. I'll close this PR and gonna open an bug for the last-rite.